src/common/containers/build-details.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import Helmet from 'react-helmet';
+
+import BuildRow from '../components/build-row';
+import BuildLog from '../components/build-log';
+import { fetchBuilds } from '../actions/snap-builds';
+
+import styles from './container.css';
+
+class BuildDetails extends Component {
+
+  componentWillMount() {
+    this.props.dispatch(fetchBuilds(this.props.fullName));
+  }
+
+  render() {
+    const { account, repo, fullName, buildId, build } = this.props;
+
+    return (
+      <div className={ styles.container }>
+        <Helmet
+          title={`${fullName} builds`}
+        />
+        {/* TODO: make into title component? */}
+        <h1>{fullName} build #{buildId}</h1>
```

Comment:
It may be worth making it a separate component (it's used in two views already), but I wasn't sure if making separate module just to render h1 is worth it ATM. Maybe if we have more complex layout or specific styles for it...